src/components/Global/DatePicker/DatePicker.jsx

```diff
@@ -0,0 +1,36 @@
+import { forwardRef } from 'react'
+import { Form } from 'react-bootstrap'
+import PropTypes from 'prop-types'
+import DatePicker from 'react-datepicker'
+import 'react-datepicker/dist/react-datepicker.css'
+
+const DatePickerComponent = ({ labelText, startDate, setStartDate }) => {
```

Comment:
```suggestion
export const DatePickerComponent = ({ labelText, startDate, setStartDate }) => {
```

I would do it this way because 
1. Default Exports are Not Discoverable .. can't be autocomplete
2. Refactoring them is difficult since they are not Discoverable.
3. while importing them Naming is a Nightmare.

Reply:
ok, let's do `export const` then.